Frontend/student-hub/src/components/Signup.js

```diff
@@ -154,24 +162,24 @@ function Signup() {
             <div style={{ backgroundColor: 'white', padding: 20, borderRadius: 10, maxWidth: 400, margin: 'auto', marginTop: '20vh' }}>
               <h2>Email Verification Code Has Been Sent To Your Email</h2>
               <form onSubmit={handleVerificationSubmit}>
-                <Grid container spacing={2}>
-                  <Grid item xs={12}>
-                    <TextField
-                      fullWidth
-                      type="text"
-                      label="Verification Code"
-                      variant="outlined"
-                      value={verificationCode}
-                      onChange={handleVerificationCodeChange}
-                      required
-                    />
-                  </Grid>
-                  <Grid item xs={12}>
-                    <Button type="submit" variant="contained" color="primary">Verify</Button>
-                  </Grid>
-                </Grid>
+              <div className="verification-container">
```

Comment:
it would be cool if we could make this into a separate component and then reuse that component within the Reset Password flow as well